appengine/mailgun/README.md

```diff
@@ -0,0 +1,62 @@
+## Mailgun on Google App Engine
+
+> [Mailgun](https://www.mailgun.com/): The Email Service For Developers
+
+### Sign up for Mailgun
+
+1. Sign up for a [Mailgun account](https://mailgun.com/signup).
+1. Add a [new domain](https://mailgun.com/app/domains).
+1. Find your API key in your new domain's settings.
+
+### Create a new Node.js app
+
+This example is going to use [Express.js](http://expressjs.com).
+
+### Configure
+
+Create an `app.yaml` in the root of your application with the following
+contents:
+
+```yaml
+runtime: nodejs
+vm: true
+api_version: 1
+env_variables:
+  PORT: 8080
+```
+
+Create a `config.js` in the root of your application with the following
```

Comment:
I'm really torn on this. We need to be consistent with the other samples, which all use environment variables.